tests/test_integration.py

```diff
@@ -229,7 +249,7 @@ def test_regular_after_current_is_supported():
     assert "--recreate" not in result.stderr
 
 
-def test_regular_after_killed_current_is_not_supported():
+def test_regular_after_killed_current_is_not_supported(print_deps_stdout_arg):
```

Comment:
Is this fixture needed here? It is not sued in the test and seems like a copy paste error.